### ISSUE TITLE
Cannot login to Discourse

### ISSUE THREAD

**Issue by waldyrious** [gobolinux/gobolinux.org#24]
I signed up for discourse on 1 April 2021, according to the confirmation email I received then. But I am not able to log in. The password I tried is rejected as invalid, and the "reset password" email never arrived. How can I resolve the situation?

**Comment by lucasvr** [comment]
I can try to send you a new invite from Discourse. Please let me know which email address you want to have the invite sent to.

**Comment by waldyrious** [comment]
Thanks for the prompt response! The email address associated with my existing account is `waldyrious@gmail.com`; if it can't be used, I might go with `waldyrious+gobolinux@gmail.com` instead.

But just to be clear, what you're suggesting would lead me to create an entirely new account? I wouldn't want to lose the username of my existing account, which matches what I use on GitHub and elsewhere ("waldyrious").

**Comment by lucasvr** [comment]
I have sent you a password reset email. If that doesn't work, then please let me know so we can recreate your account (while keeping the same username). Thanks!

**Comment by waldyrious** [comment]
It didn't work, I got no email. I checked the spam folder and still nothing. That's so weird!

**Comment by lucasvr** [comment]
That's weird indeed. I decided to delete your old account and recreate it. Please let me know if you get the invite email this time.

**Comment by waldyrious** [comment]
Still didn't get anything! I was able to create a new account using the same email and username:

![image](https://user-images.githubusercontent.com/478237/154490384-908e4f7c-c721-4046-991a-9e5d446bed32.png)

...but still got no email :( I then clicked the "Change Email Address" button and changed it to another email I have (`waldir.pimenta@gmail.com`) and it arrived immediately! It is identical to the other email I received last year, which was indeed sent to `waldyrious@gmail.com`.

Could you check if there's any filter that might be blocking emails from being sent to that address? I would like to change the email address associated with my account to it, but I don't want to be again locked out of my account...

**Comment by lucasvr** [comment]
I just checked and there's no indication that something weird happened. No filters are set, no special settings associated with your username, and I see just regular log activity. Super weird. Frankly, I've no idea what's going on. One thing that I can do, though, is to edit the primary email account associated with your username -- just in case you need to change emails again in the future (I hope that won't be needed.)

**Comment by waldyrious** [comment]
I'm afraid if you edit the email to `waldyrious@gmail.com` it might take me back to the previous issue (and also lead to me not getting email notifications, etc.)

In fact, I just tried adding `waldyrious@gmail.com` as a secondary email to my account, and didn't get the confirmation email. I also tried `waldyrious+gobolinux@gmail.com`, but the effect was the same: no email.

So maybe let's just leave it at that and assume this was some one-time glitch that hopefully won't affect other people. WDYT?
